cmd/amp-cluster/cli.go

```diff
@@ -0,0 +1,54 @@
+package main
+
+import (
+	"github.com/spf13/cobra"
+	"os"
+)
+
+const (
+	padding = 3
+)
+
+var (
+	serverAddr = ""
+	configFile = ""
+)
+
+func (g *clusterClient) cli() {
+	if err := client.init(); err != nil {
+		g.printfc(colError, "Init error: %v\n", err)
+		os.Exit(1)
+	}
+	RootCmd.PersistentFlags().BoolVarP(&g.verbose, "verbose", "v", false, `Verbose output`)
+	RootCmd.PersistentFlags().BoolVar(&g.silence, "silence", false, `No output`)
+	RootCmd.PersistentFlags().BoolVar(&g.debug, "debug", false, `display debug information`)
+	RootCmd.PersistentFlags().StringVar(&serverAddr, "server", "127.0.0.1:31315", `define the remote server address`)
+	RootCmd.PersistentFlags().StringVar(&configFile, "config", "", "Config file (default is $HOME/.amp.yaml)")
+	cobra.OnInitialize(func() {
+		g.initConfiguration(configFile, serverAddr)
+	})
+
+	// versionCmd represents the agrid version
+	versionCmd := &cobra.Command{
+		Use:   "version",
+		Short: "Display the version number of cluster-server",
+		Long:  `Display the version number of cluster-server`,
+		Run: func(cmd *cobra.Command, args []string) {
+			g.printfc(colSuccess, "amp-cluster version: %s, build: %s)\n", Version, Build)
+		},
+	}
+	RootCmd.AddCommand(versionCmd)
+
+	//Execute commad
+	cmd, _, err := RootCmd.Find(os.Args[1:])
+	if err != nil {
+		g.printfc(colError, "Arg error: %v\n", err)
```

Comment:
The pattern of `g.printfc(colError, errorDescription, err); os.exit(1)` is common enough make a `g.fatalc(errorDescription, err)` method